core/unix/os.c

```diff
@@ -8525,8 +8548,10 @@ get_dynamo_library_bounds(void)
     ASSERT(check_start == dynamo_dll_start);
     dynamo_dll_end = check_end;
 #else
-    dynamo_dll_start = check_start;
-    dynamo_dll_end = check_end;
+    if (!have_linker_vars) {
+        dynamo_dll_start = check_start;
+        dynamo_dll_end = check_end;
+    }
```

Comment:
The only other thing I would say is to document this since it relies on the user doing sthg.  However I can't find any real docs on using DR as a static lib: it's mentioned in the changelog and we added some functions in DynamoRIOConfig.cmake.in where there are some very brief docs.  If you feel up for it you could add a section to the docs on DR as a static lib which points at the config funcs and mentions the linker vars?

Reply:
Sure, I've added a TODO in my notes to write a page about this.